tasks/fh-build.js

```diff
@@ -243,174 +222,162 @@ module.exports = function(grunt) {
+          argsArray.forEach(function (arg) {
+            cmd += ' ' + arg
+          })
 
-          return cmd;
+          return cmd
         }
-      },
-      'fh-nsp': {
-        command: 'PATH=node_modules/.bin:node_modules/grunt-fh-build/node_modules/.bin:$PATH nsp check'
       }
     }
-  });
-
+  })
 
-  var checkPlaceholderFileExists = function() {
+  var checkPlaceholderFileExists = function () {
     try {
-      require(path.resolve('config/ose-placeholders.js'));
-      return true;
+      require(path.resolve('config/ose-placeholders.js'))
+      return true
+    } catch (exception) {
+      grunt.log.debug('No placeholder file found for openshift 3 - continuing normally')
+      return false
     }
-    catch (exception) {
-      grunt.log.debug('No placeholder file found for openshift 3 - continuing normally');
-      return false;
-    }
-  };
+  }
 
   var runTestsForSingleFile = function (args) {
-    var testFile = args[0];
+    var testFile = args[0]
     if (!testFile) {
-      grunt.log.errorlns("Please specify test file to run: grunt fh:testfile:filename.js");
-      return;
+      grunt.log.errorlns('Please specify test file to run: grunt fh:testfile:filename.js')
+      return
     }
     // Task name (grunt file config variable)
-    var testType = "unit_single";
+    var testType = 'unit_single'
     // Task filename parameter (used as template in runtime script)
-    grunt.config.set("unit_test_filename", testFile);
+    grunt.config.set('unit_test_filename', testFile)
     for (var index = 1; index < args.length; index++) {
-      grunt.config.set("unit_test_param" + index, args[index]);
+      grunt.config.set('unit_test_param' + index, args[index])
     }
-    var cmdArray = grunt.config.get(testType);
+    var cmdArray = grunt.config.get(testType)
     if (!cmdArray) {
-      grunt.log.errorlns("No local config for single tests. Please define " + testType + " property in config.");
-      return;
+      grunt.log.errorlns('No local config for single tests. Please define ' + testType + ' property in config.')
+      return
     }
-    grunt.task.run(['shell:fh-run-array:' + testType]);
-  };
-
-  grunt.registerTask('fh-generate-dockerised-config','Task to generate openshift config file', function() {
-    var conf = require(path.resolve('config/dev.json'));
-    var placeholders = require(path.resolve('config/ose-placeholders.js'));
-
-    grunt.log.debug('Generating openshift 3 config file');
-    _ld.forOwn(placeholders, function(value, key) {
-      grunt.log.debug('key', key, 'value', value);
-      _ld.set(conf, key, value);
-    });
-    fs.writeFileSync('conf-docker.json', JSON.stringify(conf, true, 2));
-    grunt.log.debug('Created file conf-docker.json');
-
-  });
-
+    grunt.task.run(['shell:fh-run-array:' + testType])
+  }
 
+  grunt.registerTask('fh-generate-dockerised-config', 'Task to generate openshift config file', function () {
+    var conf = require(path.resolve('config/dev.json'))
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)